Userland/Libraries/LibWeb/CSS/Screen.h

```diff
@@ -8,12 +8,13 @@
 
 #include <LibGfx/Rect.h>
 #include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/DOM/EventTarget.h>
 #include <LibWeb/Forward.h>
 #include <LibWeb/HTML/Window.h>
 
 namespace Web::CSS {
 
-class Screen final : public Bindings::PlatformObject {
+class Screen final : public DOM::EventTarget {
     WEB_PLATFORM_OBJECT(Screen, Bindings::PlatformObject);
```

Comment:
`Bindings::PlatformObject` -> `DOM::EventTarget` here, or the `Base` typedef will be wrong!